src/JiraCli.Tests/IntegrationTests/IntegrationTests.cs

```diff
@@ -77,5 +79,20 @@ private Context CreateContext(string version)
             var createStableVersionContext = CreateContext("2.0.0");
             Assert.IsTrue(await action.Execute(createStableVersionContext));
         }
+
+        
+        [TestCase("TS-1,TS-2,TS-3")]
```

Comment:
Yep - there is no test coverage for the actual argument parser though, which is where the split happens, so the way the test deals with this input is a little redundant. Still, i'll change both the test and the argument parser to add the Trim().